### PR TITLE
Fix ARM runner issues

### LINKED ISSUE #3815: Travis job for ARM is systematically failing
## Expected behavior ##

The Travis job for ARM, like all other CI jobs, passes on develop.

## Actual behavior ##

The job systematically fails with the following error:

```
____________ ERROR collecting MDAnalysisTests/lib/test_distances.py ____________

ImportError while importing test module '/home/travis/build/MDAnalysis/mdanalysis/testsuite/MDAnalysisTests/lib/test_distances.py'.

Hint: make sure your test modules/packages have valid Python names.

Traceback:

/opt/python/3.8.3/lib/python3.8/importlib/__init__.py:127: in import_module

    return _bootstrap._gcd_import(name[level:], package, level)

/home/travis/build/MDAnalysis/mdanalysis/testsuite/MDAnalysisTests/lib/test_distances.py:23: in <module>

    from tkinter import FALSE

/opt/python/3.8.3/lib/python3.8/tkinter/__init__.py:36: in <module>

    import _tkinter # If this fails your Python may not be configured for Tk

E   ModuleNotFoundError: No module named '_tkinter'
```

The issue appeared first here: <https://app.travis-ci.com/github/MDAnalysis/mdanalysis/jobs/578634668>. The corresponding commit does not look related.

## Code to reproduce the behavior ##

See https://app.travis-ci.com/github/MDAnalysis/mdanalysis/builds

## Current version of MDAnalysis ##

Develop branch on Travis CI for ARM.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #3815 

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/lib/test_distances.py [hunk 1]
```diff
@@ -20,7 +20,6 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-from tkinter import FALSE
```
Comment (IAlibay) [inline testsuite/MDAnalysisTests/lib/test_distances.py:23]: @hmacdope git blame identifies this as an addition from one of your PRs - I can't work out why it's there, is there any reason for it? (not having it fixes this issue)

**Comment by IAlibay** [comment]
Travis builds fine without the tkinter import, see: https://app.travis-ci.com/github/MDAnalysis/mdanalysis/builds/255333175

File: .cirrus.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ macos_instance:
 task:
   script: |
     brew install python@3.10
-    brew install python-tk@3.10
```
Comment (IAlibay) [inline .cirrus.yml:7]: @tylerjereddy - this shouldn't be necessary anymore (we don't import tkinter anywhere else), so I've gone ahead and removed it

**Review by jbarnoud — APPROVED** [review]
Just waiting for @hmacdope to chip in but, otherwise, it looks good to me.

File: testsuite/MDAnalysisTests/lib/test_distances.py (hunk 1 above)
Comment (hmacdope) [reply testsuite/MDAnalysisTests/lib/test_distances.py:23]: I have no idea where on earth this came from. Thanks for removing. :)

**Review by hmacdope — APPROVED** [review]

**Pull request merged**